src/app/types/components/show.ts

```diff
@@ -12,6 +12,8 @@ export type ShowActionButton = {
   name: string;
   icon?: string;
   disabled?: boolean;
+  // TODO: Remove `disabled` in favor of `show` in another PR to avoid too many items in the interface. This behavior is similar in the table.
```

Comment:
Is it really the case ? I understand that we need `show` to play or resume sessions, but having disappearing buttons could be unclear for the user.

Reply:
C'est dans la même idée que le menu dans le tableau où les éléments apparaissent ou non. Et ouais ça fait quand même beaucoup de bouton. Je mettrai une capture d'écran pour que tu puisses aussi voir avec Nicolas. Autrement, on peut mettre un menu mais c'est chelou parce que ça va cacher des quick actions. (à voir sur 2 lignes sinon ?)

Je te mets une photo asap et tu me dis.

Reply:
Avec tous les buttons

![image](https://github.com/aneoconsulting/ArmoniK.Admin.GUI/assets/45267552/a55a4708-95de-4175-8225-d232d4f4b453)